corehq/apps/analytics/management/commands/initialize_kissmetrics_plan_editions.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management import BaseCommand
+from corehq.apps.accounting.signals import subscription_upgrade_or_downgrade
+from corehq.apps.domain.models import Domain
+
+
+class Command(BaseCommand):
+    help = ("Send KISSmetrics data on subscriptions for all domains")
+
+    def handle(self, *args, **options):
+        for domain in Domain.get_all():
+            subscription_upgrade_or_downgrade.send_robust(domain, domain=domain)
```

Comment:
I agree it's redundant, but ```domain_save_callback``` takes two arguments and errors without them.  ```sender``` is not used and it doesn't appear that we follow any real convention (search for "send_robust" in the code and you'll find various things passed in for sender)